src/platform/site-wide/side-nav/index.js

```diff
@@ -0,0 +1,20 @@
+// Dependencies
+import React from 'react';
+// Relative
+import startReactApp from '../../startup/react';
+import SideNav from './components/SideNav';
+import { normalizeSideNavData } from './helpers';
+
+// Are you looking for where this is used?
+// Search for `<div data-widget-type="side-nav"></div>` to find all the places
+// this React widget is used.
+export default data => {
+  // Derive the root element to place the SideNav.
+  const root = document.querySelector(`[data-widget-type="side-nav"]`);
```

Comment:
Forgot to mention in my first review - do we need to check for this node's existence before we render?